docs/Tutorials/Visualization.md

```diff
@@ -80,3 +80,38 @@ Use the `Calculator` filter described above in "Creating Derived Volume Data"
 to create a new vector-valued dataset. Once this is created, use the `Glyph`
 filter and set the `Active Attributes` to the vector you wish to visualize.
 Make sure that `Scale Mode` is set to `vector`.
+
+### Reproducibility of Results
+
+Being able to reproduce the results of simulations is important for scientific
+integrity and accuracy. To this end, SpECTRE encodes the source tree,
+environment variables, and a record of the third party library versions into the
+executables. From most executables these can be retrieved using the flags
+`--dump-source-as filename_without_extension`, `--dump-paths`,
+`--dump-library-version`, and `--dump-environment`. The flag `--dump-only` will
+cause the program to exit after the information has been dumped. You can use the
+`--help` flag to get more details on what flags are available. The flag
+`--dump-paths` will print to screen the environment variables `PATH`, `CPATH`,
+`LD_LIBRARY_PATH`, `LIBRARY_PATH`, and `CMAKE_PREFIX_PATH` at link time. The
+flag `--dump-library-versions` will print to screen the contents of the
+`LibraryVersions.txt` file generated by CMake. The flag `--dump-environment`
+will print to screen the output of `printenv` at link time. Finally, the flag
+`--dump-source-as filename_without_extension` will dump the archive
+`filename_without_extension.tar.gz`, which contains the entire source tree at
+link time. The combined above information should be enough to reproduce
+simulations results to truncation error as long as the third party libraries
```

Comment:
I think ideally it would be roundoff, I'm not sure exactly how to quantify the errors we get by calling into system libraries. If the system library changes its algorithm, depending on what we're calling, I think we can only guarantee truncation. For example, calling Boost's ODE integration would only be guaranteed to be accurate to truncation error since the algorithm could change between versions. What are your thoughts on this and how it should be better made clear in the docs?

Reply:
You already state that this only holds if the third party libraries are unchanged.  I think that is sufficient hedging that we can claim roundoff here.

Reply:
Okay, thanks!